internal/app/server/server.go

```diff
@@ -7,8 +7,9 @@ import (
 	"github.com/envoyproxy/xds-relay/internal/pkg/log"
 
 	api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
-	"github.com/envoyproxy/go-control-plane/pkg/cache"
 	gcp "github.com/envoyproxy/go-control-plane/pkg/server"
+	internalcache "github.com/envoyproxy/xds-relay/internal/app/cache"
+	orchestrator "github.com/envoyproxy/xds-relay/internal/app/orchestrator"
```

Comment:
I would drop the name rewrites
```suggestion
	"github.com/envoyproxy/xds-relay/internal/app/cache"
	"github.com/envoyproxy/xds-relay/internal/app/orchestrator"
```